src/components/Tooltip/docs/Tooltip.md

```diff
@@ -70,7 +70,7 @@ const toggleVisibility = () => {
 
     <div style={{padding: '30px', textAlign: 'center'}}>
       <Tooltip
-        target={state.trigger === 'icon' ? <Icon name='info' /> : 
+        target={state.trigger === 'icon' ? <Icon name='info' onClick={toggleVisibility} /> : 
```

Comment:
Should we consider adding the `onClick` method to the entire Tooltip?

Reply:
In this demo, we want to hide/show the tooltip based on when the user clicks the icon or button. The wrapper includes other components that should not be click targets.

We could try to automatically pass any click handlers that are specified on the Tooltip to the Target property, but there is no guarantee it would accept an onClick handler.

When the tooltip is being used in the 'controlled input' mode as in this demo, we typically just leave full control to the engineer.